gradoop-examples/src/main/java/org/gradoop/examples/aggregation/AggregationExample.java

```diff
@@ -29,20 +30,45 @@
 
 /**
  * A self contained example on how to use the aggregate operator on Gradoop's LogicalGraph class.
- * <p>
+ *
  * The example uses the graph in dev-support/social-network.pdf
  */
 public class AggregationExample {
 
   /**
    * Path to the example data graph
    */
-  private static final String EXAMPLE_DATA_FILE = AggregationExample.class
-    .getResource("/data/gdl/sna.gdl").getFile();
+  private static final String EXAMPLE_DATA_FILE =
+    AggregationExample.class.getResource("/data/gdl/sna.gdl").getFile();
+
```

Comment:
Blank lines can be removed between String definitions (keep it consistent with other examples)